roles/core/time/molecule/default/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-centos8}-ansible:latest"
+    command: "/lib/systemd/systemd"
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
```

Comment:
I don't see this privileged in a fresh init with molecule. Why do we need it?

Reply:
To limit the scope of the extended privileges, grant SYS_ADMIN capabilities along with the same image...
So basically since many of the role are 'admin' like, and when we play with systemd and others, this is the recommended way. I can try without if needed, but that doesn't hurt anyway :-)

Reply:
Great, let's keep this. Thanks for the explanation.